rdmo/core/tests/test_utils.py

```diff
@@ -36,3 +42,49 @@ def test_join_url():
 @pytest.mark.parametrize('human,bytes', human2bytes_test_values)
 def test_human2bytes(human: Optional[str], bytes: float):
     assert human2bytes(human) == bytes
+
+
+@pytest.fixture(scope="module")
+def context(django_db_blocker):
+    """A fixture of a view context, to be used when testing 'render_to_format'.
+
+    Note: The fixture queries the database only once for the parametrized tests.
+    """
+    with django_db_blocker.unblock():
+        project = Project.objects.get(id=1)
+        project.catalog.prefetch_elements()
+        return {
+            "format": None, # will be defined in the test function
+            "project": project,
+            "project_wrapper": ProjectWrapper(project),
+            "title": project.title,
+            "resource_path": get_value_path(project)
+        }
+
+
+@pytest.mark.django_db()
+@pytest.mark.parametrize("export_format", export_formats)
+def test_render_to_format(rf, context, export_format):
```

Comment:
`render_to_format`, export_format `xml` and project don't play along. `xml` was not tested previously. This kind of data can not be exportes as xml?

Is the test still alright?

Reply:
`xml` is not using `render_to_format` so it does not need to be tested here. It should also stay in the regular tests and not be mocked.

Reply:
Looks fine otherwise, thanks a lot!

Reply:
🤦‍♂️ 

Okay, thanks for the hint. Only `pypandoc.convert_text` is mocked.

I removed multiple checks of the generated XML. Should these checks stay then?